### PR TITLE
Relax SIP fitting parameters in the tweakreg step

### PR DESCRIPTION
**Pull request by mcara**
This PR addresses reports of SIP fitting failure in the `tweakreg` step by relaxing fitting parameters and making fitting less prone to accuracy loss.

**Checklist for maintainers**
- [x] added entry in `CHANGES.rst` within the relevant release section
- [ ] updated or added relevant tests
- [ ] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [x] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

Regression tests (ALL PASSING): https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/422/

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -8,6 +8,9 @@ assign_wcs
   uncaught exception when SIP approximation fails to reach desired
   accuracy. [#7036]
 
+- Relaxed FITS WCS SIP fitting parameters for the tweakreg step to make the
```
Comment (hbushouse) [inline CHANGES.rst:11]: This code change is in the tweakreg step, so it should go in change log section for tweakreg, not assign_wcs.

File: CHANGES.rst [hunk 2]
```diff
@@ -11,6 +11,12 @@ assign_wcs
   uncaught exception when SIP approximation fails to reach desired
   accuracy. [#7036]
 
+tweakreg
```
Comment (hbushouse) [inline CHANGES.rst:14]: Needs to be moved up to the top section under the "1.7.3" section heading.

Comment (mcara) [reply CHANGES.rst:14]: I can't do this next 2 weeks. Could you please fix this?

Comment (hbushouse) [reply CHANGES.rst:14]: Will do.

**Review by hbushouse — APPROVED** [review]

**Pull request merged**